preflight-checks/nccl-group/scripts/entrypoint.py

```
@@ -0,0 +1,289 @@
+    if not pod_name:
+        log.error("POD_NAME environment variable is required")
+        return None
+
+    gang_config_dir = os.getenv("GANG_CONFIG_DIR", DEFAULT_GANG_CONFIG_DIR)
+    gang_timeout = int(os.getenv("GANG_TIMEOUT_SECONDS", str(DEFAULT_GANG_TIMEOUT)))
+
+    nprocs_env = os.getenv("NPROCS_PER_NODE", "")
+    if nprocs_env:
+        nprocs_per_node = int(nprocs_env)
+    else:
+        nprocs_per_node = _detect_gpu_count()
```

Ruff (0.15.12)

[warning] Use ternary operator `nprocs_per_node = int(nprocs_env) if nprocs_env else _detect_gpu_count()` instead of `if`-`else`-block

Replace `if`-`else`-block with `nprocs_per_node = int(nprocs_env) if nprocs_env else _detect_gpu_count()`

(SIM108)